### ISSUE TITLE
terraform apply error in 0.22.2

### ISSUE THREAD

**Issue by grpandurangipwc** [google/exposure-notifications-verification-server#1833]
### TL;DR
when we execute "terraform apply" during the deployment of 0.22.2, we getting error for Cloud Run

Error: Error updating Service "locations/us-central1/namespaces/encv-staging/services/appsync": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource appsync. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.

**Expected behavior**
No error when we run "terraform apply"

**Observed behavior**
Errored while provisioning Cloud Run services
Error: Error updating Service "locations/us-central1/namespaces/encv-staging/services/appsync": googleapi: Error 400: The feature 'sandbox selector' is not supported in the declared launch stage on resource appsync. The launch stage annotation should be specified at least as BETA. Please visit https://cloud.google.com/run/docs/troubleshooting#launch-stage-validation for in-depth troubleshooting documentation.

### Reproduction

1.  Update the version to v0.22.2 in main.tf
2.  Run terraform init, terraform plan 
3.  Run terraform apply and say yes to continue

**Screenshots**

**Environment**

- OS:
- Browser:
- Version:

**Additional information**

**Comment by sethvargo** [comment]
What does your module configuration look like?

**Comment by grpandurangipwc** [comment]
No changes from 0.21.2

**Comment by sethvargo** [comment]
Can you please share your module configuration file?

**Comment by sethvargo** [comment]
Resolved via email.
